### PR TITLE
OOP school library - add Person, Student and Teacher classess

### PR DESCRIPTION
**Pull request by Donard97**
**I created this PR to check if our project was as requested.**

_In this project, I started to build a school library app. In this milestone, I had to implement classes to represent students and teachers._

#### Created class Person with the following: ✔️ 

-  Instance vars: @id, @name, and @age
-  Constructor with name, age, and parent_permission as parameter
-  Getters and Setters
- Private method of_age? and public method can_use_services?

#### Created class Student with the following: ✔️  
- Inherits from Person
- Add @classroom to construction
- Add method play_hooky

#### Created class Teacher with the following: ✔️ 
- Inherits from Person
- Add @specialization to constructor
- Override can_use_services? so it always returns true

<hr>

_Feel free to review my project and let me know if there are any essential changes 💪 ._

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './person'
```
Comment (henatan99) [inline student.rb]: - [x] OPTIONAL: Nice work. Kindly use `require_relative` here. It is recommended to import modules in the project local directory.

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
```
Comment (henatan99) [inline teacher.rb]: - [x] OPTIONAL: Nice work. Kindly use `require_relative` here. It is recommended to import modules in the project local directory.

**Review by henatan99 — APPROVED** [review]
Hi @Donard97 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (Donard97) [reply student.rb]: Thank you for letting me know. I made the changes.

File: teacher.rb (hunk 2 above)
Comment (Donard97) [reply teacher.rb]: Thank you for letting me know. I made the changes.

**Pull request merged**